chalice/deploy/deployer.py

```diff
@@ -602,17 +602,22 @@ def _deploy_single_lambda_function(self, config, name, handler,
             function_arn = response['FunctionArn']
         else:
             self._ui.write("Creating lambda function: %s\n" % function_name)
-            function_arn = self._aws_client.create_function(
-                function_name=function_name,
-                role_arn=role_arn,
-                zip_contents=zip_contents,
-                environment_variables=config.environment_variables,
-                runtime=config.lambda_python_version,
-                handler=handler,
-                tags=config.tags,
-                timeout=self._get_lambda_timeout(config),
-                memory_size=self._get_lambda_memory_size(config),
-            )
+            kwargs = {
+                'function_name': function_name,
+                'role_arn': role_arn,
+                'zip_contents': zip_contents,
+                'environment_variables': config.environment_variables,
+                'runtime': config.lambda_python_version,
+                'handler': handler,
+                'tags': config.tags,
+                'timeout': self._get_lambda_timeout(config),
+                'memory_size': self._get_lambda_memory_size(config),
+            }  # type: Dict[str, Any]
+            if config.security_group_ids is not None:
+                kwargs['security_group_ids'] = config.security_group_ids
+            if config.security_group_ids is not None:
```

Comment:
Should this be checking config.subnet_ids instead of config.security_group_ids again?